tree-sitter-python/tree-sitter-python.cabal

```diff
@@ -17,13 +17,13 @@ library
   exposed-modules:     TreeSitter.Python
                      , TreeSitter.Python.AST
   build-depends:       base              >= 4.12 && < 5
-                     , tree-sitter       == 0.3.0.0
+                     , tree-sitter       >= 0.3
```

Comment:
I feel strongly that a version bound is not going to do the right thing here. The AST generated by `tree-sitter-python` is completely dependent on which `tree-sitter` version we pull in. That seems to thoroughly increase the risk of counterintuitive behavior (what if someone is on an old `tree-sitter`, pulls in `tree-sitter-python-0.4.0.1`, and suddenly the Hackage documentation and the data types actually generated on their system disagree? That seems to violate all sorts of versioning guarantees).

I recognize that it’s really tedious to keep bumping all these versions, but I’d rather write a `sed`/`awk` + `hackage-cli` script to do it for us, if that’s the issue. I’m proud that our [Hackage CI](https://matrix.hackage.haskell.org/#/package/tree-sitter-python) page is mostly free of mysterious

Reply:
Fair points. Gonna address this.